sys-core/server/pkg/db/db.go

```diff
@@ -13,7 +16,9 @@ import (
 var (
 	database *genji.DB
 	dbName   = "getcouragenow.db"
```

Comment:
Change to dbName   = "main.db"

Why ? because the code is not for us, but for Customers.  So "main" is nice and generic.

Reply:
Ok, Use the main.db default and put it in the .yml configuration in the future?